core/src/main/java/com/linecorp/armeria/common/DefaultRequestHeadersBuilder.java

```diff
@@ -112,4 +120,29 @@ public RequestHeadersBuilder authority(String authority) {
         setters().authority(authority);
         return this;
     }
+
+    @Override
+    @Nullable
+    public List<LanguageRange> acceptLanguages() {
+        final HttpHeadersBase getters = getters();
+        return getters != null ? getters.acceptLanguages() : null;
+    }
+
+    @Override
+    public RequestHeadersBuilder acceptLanguages(Iterable<LanguageRange> acceptLanguages) {
+        requireNonNull(acceptLanguages, "acceptLanguages");
+        final String acceptLanguagesValue = Streams
+                .stream(acceptLanguages)
+                .map(it -> (it.getWeight() == 1.0d) ? it.getRange() : it.getRange() + ";q=" + it.getWeight())
+                .collect(Collectors.joining(", "));
+        set(HttpHeaderNames.ACCEPT_LANGUAGE, acceptLanguagesValue);
```

Comment:
What would be the sensible behavior when `acceptLanguages` is empty? :thinking:

Reply:
No idea tbh.I'm inclined to say `Preconditions.checkArgument(!acceptLanguages.isEmpty, "acceptLanguages cannot be empty")`. How do you feel about that @trustin ?

Reply:
SGTM